File: Lib/Telerik/Examples/QuickStart/ChartView/NullValues/FootballTeam.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Telerik.Examples.WinControls.ChartView.NullValues
{
    public class FootballTeam
    {
        string _name;
        string _logoPath;
        int? _totalWins;
        int? _totalDraws;
        int? _totalLosses;
        List<FootballTeamStats> _stats;

        public string Name
        {
            get
            {
                return this._name;
            }
            set
            {
                this._name = value;
            }
        }

        public string LogoPath
        {
            get
            {
                return this._logoPath;
            }
            set
            {
                this._logoPath = value;
            }
        }

        public int? TotalWins
        {
            get
            {
                return this._totalWins;
            }
            set
            {
                this._totalWins = value;
            }
        }

        public int? TotalDraws
        {
            get
            {
                return this._totalDraws;
            }
            set
            {
                this._totalDraws = value;
            }
        }

        public int? TotalLosses
        {
            get
            {
                return this._totalLosses;
            }
            set
            {
                this._totalLosses = value;
            }
        }

        public List<FootballTeamStats> Stats
        {
            get
            {
                return this._stats;
            }
            set
            {
                this._stats = value;
            }
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/ChartView/NullValues/FootballTeam.vb
Imports System.Linq
Imports System.Text

Namespace Telerik.Examples.WinControls.ChartView.NullValues
	Public Class FootballTeam
		Private _name As String
		Private _logoPath As String
		Private _totalWins? As Integer
		Private _totalDraws? As Integer
		Private _totalLosses? As Integer
		Private _stats As List(Of FootballTeamStats)

		Public Property Name() As String
			Get
				Return Me._name
			End Get
			Set(ByVal value As String)
				Me._name = value
			End Set
		End Property

		Public Property LogoPath() As String
			Get
				Return Me._logoPath
			End Get
			Set(ByVal value As String)
				Me._logoPath = value
			End Set
		End Property

		Public Property TotalWins() As Integer?
			Get
				Return Me._totalWins
			End Get
			Set(ByVal value? As Integer)
				Me._totalWins = value
			End Set
		End Property

		Public Property TotalDraws() As Integer?
			Get
				Return Me._totalDraws
			End Get
			Set(ByVal value? As Integer)
				Me._totalDraws = value
			End Set
		End Property

		Public Property TotalLosses() As Integer?
			Get
				Return Me._totalLosses
			End Get
			Set(ByVal value? As Integer)
				Me._totalLosses = value
			End Set
		End Property

		Public Property Stats() As List(Of FootballTeamStats)
			Get
				Return Me._stats
			End Get
			Set(ByVal value As List(Of FootballTeamStats))
				Me._stats = value
			End Set
		End Property
	End Class
End Namespace